cafe_information

```diff
@@ -0,0 +1,21 @@
+create table CafeInSanFrancisco (
+	name varchar(32) NOT NULL,
+	address varchar(64) NOT NULL,
+	latitude double(18,14) NOT NULL,
+	longitude double(18,14) NOT NULL,
+	rating float NOT NULL,
+	Monday char(11) NOT NULL,
+	Tuesday char(11) NOT NULL,
```

Comment:
I guess we can lowercase the days too? If you have time, maybe we can rename these to better explain what value it contains - e.g. instead of 'tuesday' we can do something like 'tuesday_open_hours' or 'tuesday_hours'.

Reply:
I changed all of them :) It looks better than before